pg_search/tests/stemmer.rs

```diff
@@ -0,0 +1,244 @@
+// Copyright (c) 2023-2024 Retake, Inc.
+//
+// This file is part of ParadeDB - Postgres for Search and Analytics
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+mod fixtures;
+
+use fixtures::*;
+use pretty_assertions::assert_eq;
+use rstest::*;
+use sqlx::PgConnection;
+use tantivy::tokenizer::Language;
+
+use tokenizers::manager::language_to_str;
+
+#[rstest]
+fn language_stem_search_test(mut conn: PgConnection) {
+    // Define languages and corresponding test data
+    let languages = vec![
+        // need Language::Arabic
```

Comment:
Can you make this a TODO if it is one please? Why not add it as part of this PR?

Reply:
mark it as TODO. I use Chatgpt generate test cases,  and then find a word to search(which is come from https://snowballstem.org/demo.html). But Language Arabic can't pass test. Maybe it's basecase Arabic is right-to-left text.(Text flows from right to left)

Reply:
We have arabic tests on an existing tokenizer here: https://github.com/paradedb/paradedb/blob/dev/pg_search/tests/icu.rs#L28 Perhaps this can help you?

Reply:
add Arabic test